src/modules/piano.py

```diff
@@ -28,17 +30,29 @@ def draw(self, img):
             img = self.keys[key].draw_key(img)
         return img
 
-    def generator_7(self, spath):
+    def key_generator(self, spath, octave, key_num):

```

Comment:
good idea with key_num, but it needs to add assert or if for cases when we key_num is more than we are able to sound